rustls/src/lib.rs

```diff
@@ -401,6 +401,15 @@ mod log {
 #[macro_use]
 mod test_macros;
 
+/// This internal `atomic_sync` module aliases the `Arc` implementation to allow downstream forks
+/// of rustls targetting architectures without atomic pointers to replace the implementation
+/// with another implementation such as `portable_atomic_util::Arc` in one central location.
+mod sync {
+    // NOTE: update here may require update in `admin/use-portable-atomic-arc`.
```

Comment:
I think this comment is out of date now.

Reply:
Dang I just removed & force-pushed again thanks